static/app/components/events/eventTags/eventTagsTree.tsx

```diff
@@ -249,13 +359,22 @@ const TreeColumn = styled('div')`
 
 const TreeRow = styled('div')`
   border-radius: ${space(0.5)};
-  padding: 0 ${space(1)};
+  padding: 0 0 0 ${space(1)};
   display: grid;
   grid-column: span 2;
   grid-template-columns: subgrid;
   :nth-child(odd) {
     background-color: ${p => p.theme.backgroundSecondary};
   }
+  .invisible {
+    visibility: hidden;
+  }
+  &:hover,
+  &:active {
+    .invisible {
+      visibility: visible;
+    }
+  }
```

Comment:
Using CSS classes in combination with the `onOpenChange` event handler to determine visibility rather than doing so with mouse event listeners, seems more performant but open to alternatives.